internetarchive/archiveorg-e2e-tests
# Create e2e tests suite for archive.org

- New jobs:
  - main.yml - handles pull requests against main, pushes onto main, and cron
  - merge.yml - handles merge to main
 
- New reusable workflows:
  - books.yml
  - music.yml
  - lint-and-circular.yml
  - bookreader-e2e-tests.yml

- New tests:
  - test/books/embedded-public.book.test.ts
  - test/music/music-player-v2.test.ts
 

## Testing
- run `npm i && npm run test`
- to run individual test blocks, look at package.json & check out available scripts

- Go to Github actions board to check workflows https://github.com/internetarchive/archiveorg-e2e-tests/actions
- go to this PR's [Checks](https://github.com/internetarchive/archiveorg-e2e-tests/pull/1/checks) tab & watch tests run
